source/dyaml/test/inputoutput.d

```diff
@@ -25,10 +25,10 @@ alias std.system.endian endian;
 /// Params:  wrong = Get the incorrect BOM for this system.
 ///
 /// Returns: UTF-16 byte order mark.
-wchar bom16(bool wrong = false) pure
+wchar bom16(bool wrong = false) pure @safe
 {
-    wchar little  = *(cast(wchar*)ByteOrderMarks[BOM.UTF16LE]);
-    wchar big     = *(cast(wchar*)ByteOrderMarks[BOM.UTF16BE]);
+    wchar little = '\uFEFF';
+    wchar big = '\uFFFE';
```

Comment:
Would be cool if the usage of the global definitions would be kept.

Reply:
I could not come up with a way to convert them to wchar/dchar in a `@safe` way, unfortunately. I note that the existing global definitions exist in phobos as well, so I might clean that up in a following PR.